<launch>
	<arg 
		name="device_name"
		default="knf_simdos10_rcplus"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="address"
		default="None"/>
	<arg
		name="port"
		default="COM10"/>
	<arg
		name="switch_address"
		default="00"/>
	<arg
		name="simulation"
		default="False"/>
	<node 
		name="knf_simdos10_rcplus"
		pkg="roslabware_drivers"
		type="knf_simdos10_rcplus"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-swa $(arg switch_address)
			-l $(arg simulation)"
		output="screen"/>
</launch>